bitnami/tomcat/README.md

```diff
@@ -126,6 +126,7 @@ The following tables lists the configurable parameters of the Tomcat chart and t
 | `persistence.existingClaim` | An Existing PVC name for Tomcat volume                                                    | `nil` (uses alpha storage class annotation) |
 | `persistence.accessMode`    | PVC Access Mode for Tomcat volume                                                         | `ReadWriteOnce`                             |
```

Comment:
The option name seems to have been changed to `accessModes`, could you update this as well?